test/functional/services/failure_debugging.js

```diff
@@ -31,7 +31,7 @@ export async function FailureDebuggingProvider({ getService }) {
 
   async function onFailure(error, test) {
     // Replace characters in test names which can't be used in filenames, like *
-    const name = test.fullTitle().replace(/([^ a-zA-Z0-9/-]+)/g, '_');
+    const name = test.fullTitle().replace(/([^ a-zA-Z0-9-]+)/g, '_');
```

Comment:
is it true that `-` does not need to be escaped here?

Reply:
`-` does not need to be escaped.

I'm thinking I'd like to make a few changes to that replace.
1. the `+` makes it replace multiple instances of any of the not allowed chars with only 1 `_`.  But I think it should replace each of them with `_`.
2. I want to remove the `/`.  We do have some tests the have `/` in the describe, but I don't want that to try to split the filename into a dir/filename


const mystring = 'a!b@c#d^^^^e-f"h+j 1$2%3^4&5*6(7)8_9=0+A/B\'C';
console.log(mystring.replace(/([^ a-zA-Z0-9-])/g, '_'));

a_b_c_d____e-f_h_j 1_2_3_4_5_6_7_8_9_0_A_B_C

I'm going to make that change and push it to this PR unless I hear other suggestions.

Reply:
I guess I can't think of any cases where there's 2 chars in a row which are not valid file name characters.  I'll leave the `+`.